src/mono/wasm/debugger/BrowserDebugProxy/Firefox/FirefoxExecutionContext.cs

```diff
@@ -14,7 +14,7 @@ internal sealed class FirefoxExecutionContext : ExecutionContext
     public string? GlobalName { get; set; }
     public Result LastDebuggerAgentBufferReceived { get; set; }
 
-    public FirefoxExecutionContext(MonoSDBHelper sdbAgent, int id, string actorName) : base(sdbAgent, id, actorName)
+    public FirefoxExecutionContext(MonoSDBHelper sdbAgent, int id, string actorName, PauseOnExceptionsKind pauseOnExceptions = PauseOnExceptionsKind.Unset) : base(sdbAgent, id, actorName, pauseOnExceptions)
```

Comment:
We are not using `FirefoxExecutionContext` with non-default `pauseOnExceptions` anywhere, then maybe let's not change it yet and call instead:
```suggestion
    public FirefoxExecutionContext(MonoSDBHelper sdbAgent, int id, string actorName) : base(sdbAgent, id, actorName, PauseOnExceptionsKind.Unset)
```